test/uk/gov/hmrc/gform/sharedmodel/formtemplate/generators/FormatExprGen.scala

```diff
@@ -113,6 +120,16 @@ trait FormatExprGen {
     RoundingMode.Down,
     RoundingMode.Up
   )
+
+  private def telephoneNumberHelper(lengthRestraint: Int, phoneNumberType: PhoneNumberType) =
+    Gen.listOfN(lengthRestraint, Gen.numChar).map {
+      case phoneNumber if phoneNumberType == International => List('+') ::: phoneNumber
```

Comment:
Adding element to beginning of a list can be done like `'+' :: phoneNumber`. No need for list concatenation.